### PR TITLE
Add build packs for node v8, v10 and v12

### PR DESCRIPTION
**Pull request by webbrandon**

### CODE REVIEW

File: node/8/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM node:8
+
+ENV PHANTOMJS_VERSION 2.1.1
```
Comment (ilyakam) [inline node/8/Dockerfile:3]: Same question as in https://github.com/busbar-io/busbar-base-images/pull/6#discussion_r341437087

Comment (webbrandon) [reply node/8/Dockerfile:3]: Left comment there because Leon has personal opinion about this also.

Comment (ilyakam) [reply node/8/Dockerfile:3]: We decided to keep PhantomJS for v8.

File: node/12/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+RUN apt-get update -y && \
+      apt-get install -y \
+        jq \
+        nano \
+        htop && \
+      apt-get clean && \
+      rm -rf /var/lib/apt/lists/*
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
```
Comment (ilyakam) [inline node/12/Dockerfile:12]: Please remove the extra space at the end of this line. Depending on your editor, you may want to enable that automatically. Here's how to do that in Sublime Text 3:

```json5
    // Set to true to removing trailing white space on save
    "trim_trailing_white_space_on_save": true,
```

**Review by ilyakam — APPROVED** [review]
LGTM other than the one tiny remark inline.

**Review by waldman — DISMISSED** [review]
Can you please remove the `.busbar.yml` file?

File: node/12/Dockerfile (hunk 2 above)
Comment (webbrandon) [reply node/12/Dockerfile:12]: No.  I will need to stay conscious of it for now.  My editor has been acting up with that setting on.  Thank you for advice though, would normally agree with it.  Gonna look at using different editor soon anyways.

Comment (waldman) [reply node/12/Dockerfile:12]: >Gonna look at using different editor soon anyways.

You know... Vim is plain awesome! \o/

**Pull request merged**